packages/lib-classifier/src/store/WorkflowStepStore.spec.js

```diff
@@ -175,106 +183,61 @@ describe('Model > WorkflowStepStore', function () {
-      ROOT_STORE_INSTANCE = ROOT_STORE.create({
-        classifications: {
-          active: undefined
-        },
-        workflows: {
-          active: undefined
-        }
-      })
-      expect(ROOT_STORE_INSTANCE.workflowSteps.shouldWeShowDoneAndTalkButton).to.be.false()
+      const project = ProjectFactory.build({})
+      const panoptesClientStub = stubPanoptesJs({ workflows: workflow, subjects })
+      const rootStore = setupStores(panoptesClientStub, project)
+      expect(rootStore.workflowSteps.shouldWeShowDoneAndTalkButton).to.be.false()
     })
 
     it('should return false if there is not an active classification', function () {
-      ROOT_STORE_INSTANCE = ROOT_STORE.create({
-        classifications: {
-          active: undefined
-        },
-        workflows: {
-          active: WORKFLOW
-        }
-      })
-      expect(ROOT_STORE_INSTANCE.workflowSteps.shouldWeShowDoneAndTalkButton).to.be.false()
+      const project = ProjectFactory.build({}, { activeWorkflowId: workflow.id })
+      const panoptesClientStub = stubPanoptesJs({ workflows: workflow, subjects })
+      const rootStore = setupStores(panoptesClientStub, project, workflow)
+      expect(rootStore.workflowSteps.shouldWeShowDoneAndTalkButton).to.be.false()
     })
 
     it('should return false if not on the last step', function () {
-      ROOT_STORE_INSTANCE = ROOT_STORE.create({
-        classifications: {
-          active: classification
-        },
-        workflows: {
-          active: WORKFLOW
-        }
-      })
-      expect(ROOT_STORE_INSTANCE.workflowSteps.shouldWeShowDoneAndTalkButton).to.be.false()
+      const project = ProjectFactory.build({}, { activeWorkflowId: workflow.id })
+      const panoptesClientStub = stubPanoptesJs({ workflows: workflow, subjects })
+      const rootStore = setupStores(panoptesClientStub, project, workflow)
+      rootStore.classifications.createClassification(subject, workflow, project)
+      expect(rootStore.workflowSteps.shouldWeShowDoneAndTalkButton).to.be.false()
     })
 
     it('should return false if the workflow is not configured to hide classification summaries', function () {
-      ROOT_STORE_INSTANCE = ROOT_STORE.create({
-        classifications: {
-          active: classification
-        },
-        workflows: {
-          active: WORKFLOW
-        }
-      })
-      ROOT_STORE_INSTANCE.workflowSteps.selectStep('S2')
+      const project = ProjectFactory.build({}, { activeWorkflowId: workflow.id })
+      const panoptesClientStub = stubPanoptesJs({ workflows: workflow, subjects })
+      const rootStore = setupStores(panoptesClientStub, project, workflow)
+      rootStore.classifications.createClassification(subject, workflow, project)
+      rootStore.workflowSteps.selectStep('S2')
       // returns as falsey undefined rather than explicit false
       // this is because usually the workflow has hide_classification_summaries as undefined in the config
```

Comment:
We could explicitly set this to false in the workflow store if undefined

Reply:
Yes, I'd like to include that in a separate PR though ultimately with the goal of reverse the logic for how summaries are shown (enable them rather than disable them)